app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -56,17 +84,13 @@ def get_all_records():
     url = AIRTABLE_REQUEST_URL.format(AIRTABLE_BASE_ID)
     url = _add_fields_to_url(url)
     url += '&filterByFormula={ETL Included}=1'
-    headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
-
-    # Make request and retrieve records in json format
-    r = requests.get(url, headers=headers)
-    data = r.json()
+    data = _airtable_get_request(url)
```

Comment:
I think we should pass in headers here instead of redefining it in the airtable call and then possibly again in the error handling